### ISSUE TITLE
Incorrect caption fig 7  coordinate system section

### ISSUE THREAD

**Issue by simogasp** [scratchapixel/website#42]
In this section about the coordinate system
https://github.com/scratchapixel/website/blob/main/lessons/mathematics-physics-for-computer-graphics/geometry/coordinate-systems.md#the-right-up-and-forward-vectors
the caption of figure 7 seem to be incorrect in stating that
```
Both coordinate systems drawn in this figure are right-handed.
```
It is not the case, the second one (b) is left handed.

**Comment by AshokBatakala** [comment]
yes, that section seems to be incorrect. if any two axis are swapped the co-ordinate system changes from left-handed to right-handed.
`The only thing that defines the handedness of the coordinate system is the orientation of the left (or right) vector relative to the up and forward vectors, regardless of what these axes represent.`

this is wrong. how we label the axis , matters.
